bin/download.sunxi-fel

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2016 Konsulko Group. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+
+# The sunxi-fel tool supports passing a bus:dev to use.  However this value
```

Comment:
Yeah, I'm not thrilled with how this part works out either, as I look at the prospect of adding in PineA64 to my setup, so I would have 2 FEL boards, possibly with both having a power leak (haven't confirmed PineA64 yet).  What I've done so far, and do use for DFU testing is https://patchwork.ozlabs.org/patch/663696/ which is just a copy of the serial by-path udev rules (it's a little buggy as I see it's picked up some other devices) since at least with DFU I can pass in that directly.  I have the notion currently to use the above, add in some comments and shell my way back to a bus:dev to use.